lib/rock/webapp/tasks/api.rb

```diff
@@ -7,6 +8,9 @@ module Tasks
             class API < Grape::API
                 version 'v1', using: :header, vendor: :rock
                 format :json
+                
+                logger.formatter = GrapeLogging::Formatters::Json.new
+                use GrapeLogging::Middleware::RequestLogger, {logger: logger}
```

Comment:
Make it optional and disabled by default. And move it to the command line tool. It's not the job of an API to decide whether logging should be enabled, how it should be formatted and where it should go.